src/components/List/List.jsx

```diff
@@ -18,15 +18,17 @@ const List = ({
   id = Rivet.shortuid(),
   className,
   children,
+  plain = false,
   variant = "unordered",
   ...attrs
 }) => {
   const classes = classNames(
     inline ? `${classPrefix}-inline` : classPrefix,
-    variant === "plain" && `${classPrefix}-plain`,
-    className
+    plain ? `${classPrefix}-plain` : "",
```

Comment:
```suggestion
    (plain || variant === "plain") ? `${classPrefix}-plain` : "",
```

Would help preserve backwards compatibility if the variant plain is deprecated and not actually removed.

Reply:
We shouldn't have a "plain" prop, because then it could lead to some weird situations:

```jsx
<List plain variant="description">
```

There should just be the "variant" prop.

Reply:
Having plain as a variant is what caused the original issue (https://github.com/indiana-university/rivet-react/issues/512). While having it as a separate prop may cause strange combinations they don't directly conflict outside and it would be on the users if they want the combination and if so how to handle any strangeness.

Reply:
FYI: I added a comment to #512 that I'm hoping may inform what happens in this PR.